bucket/opentoonz.json

```diff
@@ -0,0 +1,30 @@
+{
+    "version": "1.4.0",
+    "description": "OpenToonz is a 2D animation software published by DWANGO",
+    "homepage": "https://opentoonz.github.io",
+    "license": "BSD-3-Clause",
+    "url": "https://github.com/opentoonz/opentoonz/releases/download/v1.4.0/OpenToonzSetup.exe",
+    "hash": "8b87dc4ac37a1f4acb5f4523a5db2309d2447f317cc03cd71c4c2a77e4f9cd4f",
+    "pre_install": "Write-Host 'Installing OpenToonz. This can take up to 5 minutes on an HDD.' -ForegroundColor Magenta",
+    "installer": {
+        "script": "Invoke-Expression \"$dir\\$fname /DIR=$dir /SP- /VERYSILENT /NOICONS | Out-Null \" "
+    },
+    "uninstaller": {
+        "file": "unins000.exe",
+        "args": "/VERYSILENT"
+    },
```

Comment:
Use innosetup

Reply:
I've tried with innosetup and is not working. The installer creates a regex entry and a folder in "c:/OpentoonzStuff" that is not replicable with innosetup. I will try more

Reply:
> regex entry

Does not make sense.

Then it is clear that this is not suitable for any of official buckets. If this "application" create some garbage folder named `OpentoonzStuff` on C drive it will not be accepted.